xml/templates/civicrm_sample.tpl

```diff
@@ -180,10 +180,10 @@ VALUES
 
 -- Insert sample data for event
 
-INSERT INTO `civicrm_price_set` (`name`,`title`,`is_active`,`extends`, `is_quick_config`)
+INSERT INTO `civicrm_price_set` (`name`,`title`,`is_active`,`extends`, `is_quick_config`, `financial_type_id`)
 VALUES ('rain_forest_cup_youth_soccer_tournament', 'Rain-forest Cup Youth Soccer Tournament', 1, 1, 1),
```

Comment:
won't these 2 rows need NULL or 0 values at the end otherwise column count won't match?

Reply:
ah yeah - it shouldn't be NULL either - should be a value

Reply:
OK that was it - @demeritcowboy @seamuslee001 I think it's worked now - thanks!